lib/containers/container_images.pm

```diff
@@ -110,9 +119,15 @@ sub test_opensuse_based_image {
     die 'Argument $image not provided!'   unless $image;
     die 'Argument $runtime not provided!' unless $runtime;
 
-    my ($host_version,  $host_sp,  $host_id)  = get_os_release();
-    my ($image_version, $image_sp, $image_id) = get_os_release("$runtime run --entrypoint '' $image");
-
+    my ($host_version, $host_sp, $host_id) = get_os_release();
+    my ($image_version, $image_sp, $image_id);
+    $image = 'sle15-working-container' if $runtime =~ /buildah/;
```

Comment:
Why is this hardcoded? Just pass the right image name from where you call this function.

Reply:
The reason is that this comes from as default name when buildah creates a new container. This takes part inside another subroutine. as it is now there was not 'good' way to expose it outside. Anyhow i modified it and i removed it from here. 946e17650